### PR TITLE
Bug 1884229: test: add more debug info when we fail to wait for service account

### PR DESCRIPTION
**Pull request by mfojtik**
Trying to debug cases like https://prow.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-origin-installer-e2e-azure-4.6/1311551365771694080

### CODE REVIEW

File: test/extended/util/framework.go [hunk 1]
```diff
@@ -977,18 +977,22 @@ func WaitForServiceAccount(c corev1client.ServiceAccountInterface, name string)
 			// If we can't access the service accounts, let's wait till the controller
 			// create it.
 			if kapierrs.IsNotFound(err) || kapierrs.IsForbidden(err) {
+				e2e.Logf("[DEBUG] Waiting for service account %q to be available: %v (will retry) ...", name, err)
```
Comment (deads2k) [inline test/extended/util/framework.go:980]: remove the eye catcher: `"[DEBUG]"`

Comment (mfojtik) [reply test/extended/util/framework.go]: removed

**Comment by deads2k** [comment]
/lgtm
/hold

hold for eye catcher removal.  Feel free to reapply after

**Pull request merged**